int8 motor_id